extensions/cache/src/main/resources/benchmark-redis.xml

```diff
@@ -0,0 +1,69 @@
+<benchmark xmlns="urn:radargun:benchmark:3.0">
+
+   <master bindAddress="${master.address:127.0.0.1}" port="${master.port:2103}"/>
+
+   <clusters>
+      <cluster size="8">
```

Comment:
Ok, then, please, try it out. The script runs all example benchmarks to confirm that these match the current codebase etc. I usually ran that just before a release.